plugins/woocommerce-admin/client/order-attribution/order-attribution-install-banner.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { Button, Card, CardBody } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Text } from '@woocommerce/experimental';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import OrderAttributionInstallBannerImage from './order-attribution-install-banner-image';
+import useOrderAttributionInstallBanner from './use-order-attribution-install-banner';
+import './style.scss';
+
+export const OrderAttributionInstallBanner = ( {
+	bannerImage = <OrderAttributionInstallBannerImage />,
+} ) => {
+	const {
+		isOrderAttributionInstallBannerDismissed,
+		dismissOrderAttributionInstallBanner,
```

Comment:
WDYT about shortening this for better readability?

`isDismissed`, 'dismiss'

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/51672/commits/749fb4dd2a5a4e34933b14b888380db0bc8cf3b3#diff-3815cff3f1a08b694ce40b4a0699e0cf64d9f7d0b0f3694313ae78d0bb12f56bR21.